src/app/app.component.html

```diff
@@ -21,11 +21,29 @@
                 </li>
             </ul>
         </div>
+
+        <!-- nav bar links after login -->
+        <div class="collapse navbar-collapse justify-content-end" id="navbarNav" *ngIf="router.url == '/dashboard' || router.url == '/group'">
+            <ul class="navbar-nav">
+                <li class="nav-item ">
+                    <div class="dropdown">
+                        <button class="btn text-light dropdown-toggle" type="button" id="profileDropdown" data-bs-toggle="dropdown">
+                    Dropdown button
+                  </button>
+                        <ul class="dropdown-menu">
+                            <li><a class="dropdown-item" href="#">Profile</a></li>
+                            <li><a class="dropdown-item" routerLink="/">Logout</a></li>
+                        </ul>
```

Comment:
For profile & logout option, create an array in component and use *ngFor in html